hosted/preparation/squid.conf

```diff
@@ -0,0 +1,82 @@
+#
```

Comment:
I made a copy of the original file in `hosted/preparation/squid.conf` because  the original location was harder to access  , after moving the code to root dir I can access it easily again. I reverted back to original location.